### ISSUE TITLE
Problems with training the Views Predictor

### ISSUE THREAD

**Issue by zhang0931** [pse-ecn/pose-sensitive-embedding#12]
hi,

Thank the author, you job is so great. I successfully trained once,  but there was a problem when I trained again .The problem is shown in the screenshot
I execute the following:
sudo python3 trainer_views.py --output=./output/Views_mode_new --data=/data/zhangli/datasets/RAP_dataset --dataset-name=rap --batch-size=16 --num-epochs=100 --network-name=resnet_v1_50_views --initial-checkpoint=/data/zhangli/pose-sensitive-embedding-master/checkpoint_dir/resnet_v1_50.ckpt --checkpoint-exclude-scopes=resnet_v1_50/logits --trainable-scopes=resnet_v1_50/3Views

![image](https://user-images.githubusercontent.com/31616306/44081268-e5a99458-9fe0-11e8-8a0f-496ade0469bb.png)
![image](https://user-images.githubusercontent.com/31616306/44081313-03128a7c-9fe1-11e8-83dc-f3017b582466.png)
![image](https://user-images.githubusercontent.com/31616306/44081347-196f9170-9fe1-11e8-8cf6-480e709f9179.png)
![image](https://user-images.githubusercontent.com/31616306/44081376-2d0bfa2a-9fe1-11e8-8e59-68d211044d3e.png)
![image](https://user-images.githubusercontent.com/31616306/44081428-44d2a258-9fe1-11e8-94c0-94435dc6ed3c.png)

**Comment by andreas-repp** [comment]
Try running it with `--ignore-missing-variables`. Be aware that this randomly initializes the variables not available in the checkpoint you provide (the variables listed in the error message).

**Comment by SpiritBear000** [comment]
hi,
I execute the following:
python3 trainer_views.py --output=everything_generate/output_dir --data=/media/lab210/Work/WH_HOME/SubjectDownload/Project/DATASETS/_datasets_/RAP/RAP_dataset --dataset-name=rap --batch-size=16 --num-epochs=100 --network-name=resnet_v1_50_views --initial-checkpoint=/media/lab210/Work/WH_HOME/SubjectDownload/Project/reid/pose-sensitive-embedding/everything_generate/checkpoint/resnet_v1_50.ckpt --checkpoint-exclude-scopes=everything_generate/resnet_v1_50/logits --trainable-scopes=resnet_v1_50/3Views --ignore-missing-variables

but get the error:
![2018-10-02 12-30-43](https://user-images.githubusercontent.com/32519071/46329309-0ab9fb00-c63f-11e8-8203-2b36c78afc30.png)

The 'resnet_v1_50.ckpt' was downloaded from 'https://github.com/tensorflow/models/tree/master/research/slim'

------------
And another question:

--checkpoint-exclude-scopes=everything_generate/resnet_v1_50/logits

where is the logits?

**Comment by ssarfraz** [comment]
try --checkpoint-exclude-scopes= resnet_v1_50/logits  i.e. with only the network name, dont need to append the path

**Comment by SpiritBear000** [comment]
Thank you for your help!

在 2018-11-01 02:21:46，ssarfraz <notifications@github.com> 写道：


try --checkpoint-exclude-scopes= resnet_v1_50/logits i.e. with only the network name, dont need to append the path
